packages/grid/x-data-grid/src/components/GridRow.tsx

```diff
@@ -588,128 +588,394 @@ GridRow.propTypes = {
   onMouseLeave: PropTypes.func,
   pinnedColumns: PropTypes.shape({
```

Comment:
We could probably add this prop to the one to skip :see_no_evil:

Reply:
Yes, absolutely 👍🏻